js/stores/eventStore.js

```diff
@@ -11,7 +11,8 @@ const { isSourceAboutUrl } = require('../lib/appUrlUtil')
 
 let eventState = Immutable.fromJS({
   page_load: [],
-  page_view: []
+  page_view: [],
+  publisher_map: {}
```

Comment:
not sure if this is the best way to store it because we may want to consider things like embedded youtube videos